java/integTest/src/test/java/glide/standalone/TransactionTests.java

```diff
@@ -307,6 +309,67 @@ public void watch() {
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
 
+    @Test
+    @SneakyThrows
+    public void watch_binary() {
+        GlideString key1 = GlideString.gs("{key}-1" + UUID.randomUUID());
+        GlideString key2 = GlideString.gs("{key}-2" + UUID.randomUUID());
+        GlideString key3 = GlideString.gs("{key}-3" + UUID.randomUUID());
+        GlideString key4 = GlideString.gs("{key}-4" + UUID.randomUUID());
```

Comment:
Import `gs` and shorten all calls
```suggestion
        GlideString key1 = gs("{key}-1" + UUID.randomUUID());
        GlideString key2 = gs("{key}-2" + UUID.randomUUID());
        GlideString key3 = gs("{key}-3" + UUID.randomUUID());
        GlideString key4 = gs("{key}-4" + UUID.randomUUID());
```
Apply thoughout
See `SharedCommandTests.java` for reference